handlers.go

```diff
@@ -75,6 +78,8 @@ func (s *server) authenticate(w http.ResponseWriter, r *http.Request) {
 		logger.Errorf("Failed to save state in store: %v", err)
 		returnStatus(w, http.StatusInternalServerError, "Failed to save state in store.")
 		return
+	} else {
```

Comment:
Same, is else needed here?

Reply:
This also logs the struct and means it can be seen what has changed in since the struct was created.